CONTRIBUTING.md

```diff
@@ -20,12 +20,29 @@ To check your the project's dependencies run:
 $ scripts/check_setup.sh
 ```
 
-Write some features.
+## Build the Unity Buy SDK
+
+Under the hood, the Unity Buy SDK works on top of GraphQL. In order to query Shopify, we generate a GraphQL client based on the [Storefront API](https://help.shopify.com/api-storefront-api) introspection schema.
+
+To build the GraphQL client, run the following command:
+```bash
+$ scripts/build.sh
+```
+
+Now you are free to write some features.
```

Comment:
Nit: I think phrasing this as a prerequisite might sound better. For example, removing this and adding something like `Before you can start developing on the SDK you'll need to run the code generator. Under the hood,...` above.